src/qt/bitcoingui.cpp

```diff
@@ -396,6 +396,8 @@ void BitcoinGUI::createActions()
                     connect(activity, &OpenWalletActivity::opened, this, &BitcoinGUI::setCurrentWallet);
                     connect(activity, &OpenWalletActivity::finished, activity, &QObject::deleteLater);
                     connect(activity, &OpenWalletActivity::finished, dialog, &QObject::deleteLater);
+                    bool invoked = QMetaObject::invokeMethod(activity, "open");
+                    assert(invoked);
```

Comment:
When can this fail?

Reply:
If the slot is renamed it would still compile. With Qt 5.10 it is possible to invoke like `QMetaObject::invokeMethod(activity, &OpenWalletActivity::open).

Reply:
Thanks!